src/deprecated.jl

```diff
@@ -1,5 +1,10 @@
 import Base: @deprecate
 
+function Base.map(f::Function, sdf::SubDataFrame)
+    Base.depwarn("Calling map on SuBDataFrame is deprecated", :map)
```

Comment:
Erroneously capitalized B there. Also, could this not just be `@deprecate map(f::Function, sdf::SubDataFrame) f(sdf)`?

Reply:
Actually now - after looking at #1520 I can see that @nalimilan included this deprecation there so I will remove it from this PR.

Reply:
Better keep the function here then, and remove it when adding the deprecation at #1520.

Reply:
This is what I did if I understand you correctly

Reply:
What I mean is that you shouldn't remove this line: https://github.com/JuliaData/DataFrames.jl/pull/1588/files#diff-7b74440e6f375ca303e4c5ef15117c26L220

Reply:
Right - sorry for forgetting this. Fixed